department-of-veterans-affairs/va.gov-cms
# VACMS-7068 claro login page

## Description

Relates to #7068. Also includes a fix for vagovclaro to work with the updated helptext fix. :)

This is another iteration on the new login experience... there will likely be another iteration after design/UX have a chance to look at it. 

Code in this PR located in `vagovclaro.theme` could probably move to the `va_gov_help_center` or `va_gov_login` modules when this theme is turned on, but I kept it in the theme for now to prevent changing the current experience... happy to make those updates there now (PIV / Get help) if that is appropriate.

## Testing done
Flipped `activate: true` in `config/local/simplesamlphp_auth.settings.yml` in order to see the PIV login locally. Otherwise, looked at the login page with site alert turned on and off.

## Screenshots
<img width="1913" alt="Screen Shot 2021-12-15 at 1 33 43 PM" src="https://user-images.githubusercontent.com/11279744/146269186-559b33dd-ed72-4433-a0d6-cae4cb64fe4f.png">

## QA steps
1. enable vagovclaro as the default theme
2. log out to see the new login experience
3. ???

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

No announcement necessary until the turn-on date is closer at hand.
